pkg/resource/grafana-datasource/reconciler.go

```diff
@@ -28,6 +28,10 @@ func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Inter
 	logger := log.FromContext(ctx)
 	logger.Info("grafanadatasource create")
 
+	if lc.IsCAPI() {
+		return r.ReconcileDelete(ctx, lc)
+	}
```

Comment:
Why do we delete datasource on CAPI only ? Please add a comment on this.

Reply:
It is written in the PR description

Reply:
```suggestion
        // We delete the datasource only in CAPI as the datasource is now managed by the observability-operator
	if lc.IsCAPI() {
		return r.ReconcileDelete(ctx, lc)
	}
```

Reply:
How does this works when 2 operators are reconciling the same datasource ? Isn't there a conflict here where one operator would delete the resource while the other creates it ?

Reply:
No because the logging operator creates a secret that is picled up by grafana provisionning and thé datasource IS named Loki


Olly OP creates the datasource via an API call and thé ds will bé named Loki Olly OP for now

Reply:
So once this is released we can rename the datasource in olly-op right ? Please make sure we have a todo their then.